### PR TITLE
Update Documentation for Minikube and AWS

### PR DESCRIPTION
**Pull request by leiarenee**
- Updated documentation for minikube([issue 1044](https://app.zenhub.com/workspaces/dev-5e7e84dddc966f05a627a19b/issues/gh/redkubes/otomi-core/1044))
- Updated Documentation for AWS
- Changed k8s version defaults to 1.23

### CODE REVIEW

**Comment by leiarenee** [comment]
The command line under test
```bash
helm install otomi otomi/otomi --set cluster.k8sVersion="1.23" --set cluster.name=minikube --set cluster.provider=custom --set apps.host-mods.enabled=false --set apps.metrics-server.extraArgs.kubelet-insecure-tls=true --set apps.metrics-server.extraArgs.kubelet-preferred-address-types=InternalIP
```

**Comment by leiarenee** [comment]
Metrics server installed successfully. On the other hand we have other problems.

Keycloak fails.

**Comment by leiarenee** [comment]
See [Quick Start Minikube for changes](https://github.com/redkubes/quickstart/tree/update-aws-install/minikube)

File: minikube/README.md [hunk 1]
```diff
@@ -1,61 +1,161 @@
...
-# Configure metallb with the IP as seen in the figure below
+```
+
+Terminal Output
+
+```
+192.168.49.2
+```
+
+_Pleas note that IP might and will probably be different in your system_
```
Comment (j-zimnowoda) [inline minikube/README.md:102]: _Please

File: minikube/README.md [hunk 2]
```diff
@@ -1,61 +1,161 @@
...
 ```
 
+Terminal Output
+
+```
+-- Enter Load Balancer Start IP: 192.168.49.101
+-- Enter Load Balancer END IP: 192.168.49.121
+```
+
+Define the load balancer's ip range using the above example replacing the IP address which is acquired in Step 3.
```
Comment (j-zimnowoda) [inline minikube/README.md:117]: IP address

**Comment by leiarenee** [comment]
Spell check corrections done.

**Review by j-zimnowoda — APPROVED** [review]

**Pull request merged**